### ISSUE TITLE
SSL error coming from censys internals

### ISSUE THREAD

**Issue by konklone** [Censys/censys-python#13]
With censys 0.0.6 and requests 2.11.1, I get a super strange SSL error:

```
  File "[localpath]/censys.py", line 35, in gather
    certificate_api = certificates.CensysCertificates(uid, api_key)
  File "/home/eric/.pyenv/versions/3.5.2/lib/python3.5/site-packages/censys/base.py", line 112, in __init__
    CensysAPIBase.__init__(self, *args, **kwargs)
  File "/home/eric/.pyenv/versions/3.5.2/lib/python3.5/site-packages/censys/base.py", line 59, in __init__
    self.account()
  File "/home/eric/.pyenv/versions/3.5.2/lib/python3.5/site-packages/censys/base.py", line 104, in account
    return self._get("account")
  File "/home/eric/.pyenv/versions/3.5.2/lib/python3.5/site-packages/censys/base.py", line 95, in _get
    return self._make_call(self._session.get, endpoint, args)
  File "/home/eric/.pyenv/versions/3.5.2/lib/python3.5/site-packages/censys/base.py", line 76, in _make_call
    res = method(url, params=args)
  File "/home/eric/.pyenv/versions/3.5.2/lib/python3.5/site-packages/requests/sessions.py", line 487, in get
    return self.request('GET', url, **kwargs)
  File "/home/eric/.pyenv/versions/3.5.2/lib/python3.5/site-packages/requests/sessions.py", line 475, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/eric/.pyenv/versions/3.5.2/lib/python3.5/site-packages/requests/sessions.py", line 585, in send
    r = adapter.send(request, **kwargs)
  File "/home/eric/.pyenv/versions/3.5.2/lib/python3.5/site-packages/requests/adapters.py", line 477, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: EOF occurred in violation of protocol (_ssl.c:645)
```

I am not yet sure of the root cause, but wanted to open the issue while I had the information.

**Comment by zakird** [comment]
Thanks for the issue. This should now be resolved.

Google has been deprecating non-SNI-based TLS over the past several months. When we replaced our certificate yesterday, we unexpected lost this non-SNI access into Censys, which (long story short) broke www.censys.io. This is now working again. I ran the tests in the library and they seem to be fine, but let us know if you're still running into problems.
